src/components/ConnectorName.tsx

```diff
@@ -15,6 +15,14 @@ function ConnectorName({ connector, iconPath, iconSize }: Props) {
         typeof connector === 'string' ? connector : getConnectorName(connector)
     );
 
+    const connectorIcon = useConstant(() =>
+        typeof iconPath === 'string'
+            ? iconPath
+            : typeof connector !== 'string'
+            ? getConnectorIcon(connector)
+            : undefined
+    );
```

Comment:
Let's you just pass the open graph object in the connector prop and it handles fetching the name and icon. Icon will default to nothing.